### ISSUE TITLE
Do you have any plans to bring your extension to other web browsers?

### ISSUE THREAD

**Issue by tobiaalberti** [sienori/Tab-Session-Manager#166]
Hello,

For various reasons I'm currently using **Google Chrome** as my primary web browser and I would very much like being able to use **Tab Session Manager** there.
Is bringing it to other browsers (like Chrome, **Edge**, and **Opera**) something you're interested in or planning to do?

Regards.

**Comment by hdf** [comment]
It would be problematic, [as Firefox uses promises, and Chrome uses callbacks in its API](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Chrome_incompatibilities).
Opera is based on Chromium, so if the extension works on Chrome, it should also work on Opera.
As for Edge, have look at this [beauty](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Browser_support_for_JavaScript_APIs).
There is also [this tool](https://www.microsoft.com/en-GB/store/p/microsoft-edge-extension-toolkit/9nblggh4txvb?rtc=1), to help port a Chrome extension to Edge.

Basically, it seems to be easier to port from Chrome to other places, than from Firefox to Chrome.

May I suggest [Session Buddy](https://chrome.google.com/webstore/detail/session-buddy/edacconmaakjimmfgnblocblbcdcpbko) as a Chrome alternative?

**Comment by tobiaalberti** [comment]
Thank you for your detailed response.
I'm not a technical person, but I think I generally get what you're saying.

I heard about your extension on a _Security Now_ (a _TWiT_ podcast) episode from a few weeks ago. Apparently _Steve Gibson_ is a user.

Thank you very much for your suggestion.
I've been testing out _Session Buddy_ for the pas hour or so and I think it's pretty perfect for my current needs.

Have a nice day/night.

**Comment by sienori** [comment]
It is hard to verify with multiple browsers every time I add features, so I can not do it now.
In the future I may develop it.
